Core/Src/gw_flash.c

```diff
@@ -252,6 +252,10 @@ void OSPI_Program(OSPI_HandleTypeDef *hospi, uint32_t address, const uint8_t *bu
   unsigned iterations = buffer_size / 256;
```

Comment:
This should be 

`unsigned iterations = (buffer_size + 255) / 256;`

That way it will handle writes that are not aligned with 256 bytes. A few examples:

- buffer_size = 16; => 1 iteration
- buffer_size = 256; => 1 iteration
- buffer_size = 257; => 2 iterations

My change in https://github.com/kbeckmann/game-and-watch-retro-go/commit/306b4f2479e2e1ad2b7f62c2a0741262bf65c182#diff-92ea044cbdad0434abccdddeb2d1fb45316ff2734a4837f344f267c01d374702L248-L257 addressed this differently, but this way will also work.